### ISSUE TITLE
Test For WordPress Coding Standards

### ISSUE THREAD

**Issue by ataylorme** [pantheon-systems/WordPress#202]
`wp-config.php` and `wp-content/mu-plugins/pantheon` should adhere to WordPress coding standards. An automated test should also run on every pull request.

Related: #179

**Comment by greg-1-anderson** [comment]
Coding standards are cool, but I am concerned about how many merge conflicts we might cause in downstream repositories if we reformatted wp-config.php.

**Comment by dylanjameswagner** [comment]
There is an increased possibility for merge conflicts the moment anyone edits this file because it does not implicitly follow any coding standard. I routinely add at least four customizations that cannot be done anywhere else.

**Comment by ataylorme** [comment]
Perhaps this issue will morph into creating a `wp-config.pantheon.php` and conditionally requiring that from `wp-config.php`

**Comment by greg-1-anderson** [comment]
Yes, Pantheon made a mistake a long time ago and did not set up wp-config.php to be particularly maintainable. The result is that Pantheon users have to modify wp-config.php, so any time we make modifications to it, we risk creating a lot of merge conflicts.

Fixing this problem will cause a lot of merge conflicts, but perhaps it will be worth it in the long run.

**Comment by pwtyler** [comment]
wp-config-pantheon.php exists now, and coding standards are followed for mu plugin.
